### DISCUSSION TITLE
Elastic Security in Kibana

### DISCUSSION THREAD

**Discussion by mkay2999** [Security-Onion-Solutions/securityonion#10593] [2.4] [answered]
any plans to add Elastic Security into kibana? 

I see that a couple weeks ago you guys added support for the elastic defend integration for the agents, but security is disabled on the navigation bar and when i navigate to it from the search it get an error about privileges being required. 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/94582994/00c4b31c-7c8f-4349-ab23-466850b9103e)

I've searched the documentation and cant find any way to turn this feature on, any assistance would be appreciated.

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.4/kibana.html#features:
> You can enable or disable specific features by clicking the main menu in the upper left corner, then click Stack Management, then click Spaces, then click Default. For more information, please see https://www.elastic.co/guide/en/kibana/master/xpack-spaces.html#spaces-control-feature-visibility.

**Reply by mkay2999** [reply]
perfect, thank you
